<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Archivo</class>
 <widget class="QGroupBox" name="Archivo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>420</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>270</width>
    <height>420</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>270</width>
    <height>420</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>270</width>
    <height>420</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Jura</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>GroupBox</string>
  </property>
  <property name="title">
   <string>Grabaci√≥n</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre del archivo:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="TextFileName">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>271</width>
     <height>27</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>90</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Bytes grabados:</string>
   </property>
  </widget>
  <widget class="QLabel" name="LabelBytesGrabados">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>110</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>0 KB </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
